addons/godottpd/http_response.gd

```diff
@@ -44,24 +44,24 @@ func send_raw(status_code: int, data: PoolByteArray = [], content_type: String =
 # - data: The body data to send []
 # - content_type: The type of the content to send ["text/html"]
 func send(status_code: int, data: String = "", content_type = "text/html") -> void:
-	send_raw(status_code, data.to_ascii(), content_type)
+	send_raw(status_code, data.to_ascii_buffer(), content_type)
 
 # Send out a JSON response to the client
-# This function will internally call the `send()` method 
+# This function will internally call the `send()` method
 #
 # #### Parameters
 # - status_code: The HTTP status code to send
 # - data: The body data to send, must be a Dictionary or an Array
 func json(status_code: int, data) -> void:
-	send(status_code, JSON.print(data), "application/json")
+	send(status_code, JSON.stringify(data), "application/json")
 
 
 # Sets the response’s header "field" to "value"
 #
 # #### Parameters
 # - field: the name of the header i.e. "Accept-Type"
 # - value: the value of this header i.e. "application/json"
-func set(field: String, value: String) -> void:
+func set(field: StringName, value: Variant) -> void:
```

Comment:
Why Variant? Can a http header be something else than a string?

Reply:
the "set" function of this class is overriding the "set" function from the class "Object"

Reply:
Ohhh!